src/components/PageLayout/StromLogo/StromLogo.tsx

```diff
@@ -3,32 +3,31 @@ import {FC} from 'react'
 
 import Strom from '@/svg/strom.svg'
 
+import {HeaderHeightContainer} from '../Header/HeaderHeightContainer'
+
 export const StromLogo: FC = () => {
+  const {height} = HeaderHeightContainer.useContainer()
+
   return (
     <Box
       sx={{
+        // NOTE (odstranit): 'sticky' (a bez width) ma inak tiez celkom zaujimavy vysledok
```

Comment:
sticky sa lepsie sprava pri refreshi, kde ma vychodziu poziciu podla normalneho layout algoritmu, az pri scrolle sa oddeli. ale to by snad slo nejak vyriesit aj tak, nehral som sa s tym moc dlho. mozno nakoniec `position: 'absolute'` voci parentovi...

Reply:
docs (linkujem vsade vsetko useful co mam otvorene): https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky